### ISSUE TITLE
Packages error.

### ISSUE THREAD

**Issue by dsx12** [Screetsec/TheFatRat#330]
The packages "dx", "appt", "apktool" are not downloading from the fatrat "./setup.sh" script. I attempted to install the packages manually, after installing I received the same errors. Does anyone know how to fix this?

![screenshot from 2018-11-21 21-15-56](https://user-images.githubusercontent.com/45248846/48878011-a95b1080-edd3-11e8-9a7b-2c8430921bc7.png)

**Comment by dsx12** [comment]
Here's my install log:
xterm -> OK
Dns-Utils -> OK
GCC -> OK
Apache2 -> OK
Gnome Terminal -> OK
UPX -> OK
Ruby -> OK
Openssl -> OK
Jarsigner -> OK
Unzip -> OK
Keytool -> OK
Mingw64 -> OK
Mingw32 -> OK
dx -> Not OK
aapt -> Not OK
apktool -> Not OK
Dex2Jar -> OK
Metasploit -> OK
Backdoor-Factory -> OK
Searchsploit -> OK
Was not possible to install The Packages Labeled (Not Ok) in this list above
Try : (apt-get remove --purge <packagename> && apt-get autoremove && apt-get install -f)
before running fatrat setup script again

**Comment by dsx12** [comment]
I've read other comments posts on this subject with no solution. I have updated my sources.list file to no avail. I searched "find /usr/ -name aapt" and "find /usr/ -name dx" afterwards my terminal outputted "/usr/local/sbin/aapt/", "/usr/local/sbin/dx"/. I deleted the files outputted and still nothing. Tried removing the files also tried "rm -f /var/lib/dpkg/lock".

**Comment by dsx12** [comment]
I also tried uninstalling the packages listed as (Not ok). And it still won't install them from setup.sh. If I install them manually they are printed in red as a error etc.

**Comment by peterpt** [comment]
put /usr/local/sbin in your system path

use a text editor and add that path to your bashrc file in your user folder

nano ~/.bashrc
add this line
export PATH="$PATH:/usr/local/sbin"

Save it , close your terminal and reopen it again .

**Comment by ssayezzz** [comment]
same issue is going with me too.

**Comment by peterpt** [comment]
dont wake up the dead people .
Why you all always want to revive old posts ?!!!
that is a bad habit .
before i lock up this thread :
Here it is the thing , those tools are inside fatrat folders , do not install them directly from apt because when you install fatrat then those version you installed before will be removed .
This happens because fatrat uses the latest versions available .
What fatrat does is to symlink the tools from its tools folders to your "/usr/local/sbin" .
This means that after installing fatrat when you write apktool , then you will open the apktool binary inside fatrat folder , because the symlink inside your /usr/local/sbin folder will reeedirect your request to fatrat folder .
Now , if you dont have "/usr/local/sbin" folder in your path variables , then no matter how many times fatrat tries to install the tool it will never work .
basically what is needed is to add to $HOME/.bashrc   file the prvios line i wrote before , witch is :
export PATH=$PATH:/usr/local/sbin

and then close your terminal and open it again to load the new config .
